tests/test_diffraction_objects.py

```diff
@@ -702,3 +702,66 @@ def test_copy_object(do_minimal):
     do_copy = do.copy()
     assert do == do_copy
     assert id(do) != id(do_copy)
+
+
+@pytest.mark.parametrize(
+    "starting_all_arrays, scalar_to_add, expected_all_arrays",
+    [
+        # Test scalar addition to xarray values (q, tth, d) and expect no change to yarray values
```

Comment:
the add operation is supposed to operate on the yarray leaving xarrays unaffected.